### PR TITLE
K8s 1.22 update

### PR DESCRIPTION
**Pull request by heronrs**
### What is the feature/fix?

Updates k8s to 1.22 across all cloud providers.

### Add screenshot or video (optional)

** Any screenshot or video capture using the feature **

### Does it has a breaking change?

Yeah, cloud providers don't allow downgrading k8s versions.

### How to use/test it?

All current features should still work as expected

### Checklist
- [ ] New coverage tests
- [ ] Unit tests passing
- [ ] E2E tests passing
- [ ] E2E downgrade/update test passing
- [ ] Documentation updated
- [ ] No warnings or errors on Deepsource/Codecov

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,7 +54,7 @@ package:
 
 release:
 	test -n "$(VERSION)" # VERSION
-	git tag $(VERSION) -m $(VERSION)
+	git tag $(VERSION) -s -m $(VERSION)
```
Comment (heronrs) [inline Makefile:57]: just sign the commit by default

File: terraform/router/nginx/main.tf [hunk 2]
```diff
@@ -225,6 +225,12 @@ resource "kubernetes_deployment" "ingress-nginx" {
         service_account_name             = "ingress-nginx"
         automount_service_account_token  = true
         priority_class_name              = var.set_priority_class ? "system-cluster-critical" : null
+        security_context {
```
Comment (heronrs) [inline terraform/router/nginx/main.tf:228]: I missed this part on #512

**Review by Twsouza — APPROVED** [review]
LGTM

**Review by nightfury1204 — APPROVED** [review]
:fire:

**Review by yuriadams — DISMISSED** [review]

**Pull request merged**